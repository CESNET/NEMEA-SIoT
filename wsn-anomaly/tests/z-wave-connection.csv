double ACKCount,double ACKWaiting,double CANCount,uint64 INCIDENT_DEV_ADDR,double NAKCount,double OOFCount,double SOFCount,time TIME,double averageRequestRTT,double averageResponseRTT,double badChecksum,double badroutes,double broadcastReadCount,double broadcastWriteCount,double callbacks,double dropped,double lastRequestRTT,double lastResponseRTT,double netBusy,double noACK,double nonDelivery,double notIdle,double quality,double readAborts,double readCount,double receiveDuplications,double receiveUnsolicited,double receivedCount,double retries,double routedBusy,double sentCount,double sentFailed,double writeCount
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:21.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:21.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:21.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:21.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:26.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:26.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:26.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:26.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:31.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:31.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:31.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:31.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:36.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:36.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:36.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:36.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:41.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:41.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:41.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:41.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:46.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:46.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:46.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:46.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:51.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:51.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:51.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:51.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:17:56.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:17:56.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:17:56.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:17:56.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:01.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:01.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:01.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:01.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:06.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:06.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:06.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:06.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:11.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,0.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:11.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:11.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:11.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:16.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,2.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:16.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:16.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,36.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:16.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:21.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,2.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:21.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:21.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,136.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,33,0.000000,0.000000,0.000000,2018-04-05T15:18:21.000,0.000000,4.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,4.000000,3.000000,0.000000
31.000000,0.000000,0.000000,1113,0.000000,57.000000,57.000000,2018-04-05T15:18:26.000,0.000000,0.000000,0.000000,0.000000,0.000000,9.000000,0.000000,2.000000,0.000000,0.000000,0.000000,3.000000,0.000000,0.000000,0.000000,0.000000,57.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,31.000000
0.000000,1522941431.000000,0.000000,11,0.000000,0.000000,0.000000,2018-04-05T15:18:26.000,0.000000,2.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,2.000000,0.000000,0.000000
0.000000,1522941431.000000,0.000000,22,0.000000,0.000000,0.000000,2018-04-05T15:18:26.000,25.000000,16.000000,4.000000,3.000000,0.000000,0.000000,0.000000,0.000000,26.000000,236.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,0.000000,10.000000,0.000000,0.000000,16.000000,0.000000,0.000000
