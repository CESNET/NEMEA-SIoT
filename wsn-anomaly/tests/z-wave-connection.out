2,1522941531.000000,0.071429,0.000000,"Lower grow limit","average","receivedCount"
2,1522941531.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941531.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941536.000000,0.066667,0.000000,"Lower grow limit","average","receivedCount"
2,1522941536.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941536.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941541.000000,0.062500,0.000000,"Lower grow limit","average","receivedCount"
2,1522941541.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941541.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941546.000000,0.058824,0.000000,"Lower grow limit","average","receivedCount"
2,1522941546.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941546.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941551.000000,0.055556,0.000000,"Lower grow limit","average","receivedCount"
2,1522941551.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941551.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941556.000000,0.052632,0.000000,"Lower grow limit","average","receivedCount"
2,1522941556.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941556.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941561.000000,0.050000,0.000000,"Lower grow limit","average","receivedCount"
2,1522941561.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941561.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941566.000000,0.047619,0.000000,"Lower grow limit","average","receivedCount"
2,1522941566.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941566.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941571.000000,0.045455,0.000000,"Lower grow limit","average","receivedCount"
2,1522941571.000000,0.100000,0.000000,"Lower grow limit","moving_average","receivedCount"
2,1522941571.000000,0.100000,0.000000,"Lower grow limit","moving_variance","receivedCount"
2,1522941576.000000,0.173913,0.000000,"Lower grow limit","average","receivedCount"
2,1522941586.000000,8.900000,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941591.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941596.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941601.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941606.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941611.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
1113,1522941616.000000,23.000000,0.000000,"Higher hard limit","moving_median","dropped"
2,1522941616.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941621.000000,8.666667,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941626.000000,8.900000,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941631.000000,6.622222,0.000000,"Higher grow limit","moving_variance","receivedCount"
2,1522941681.000000,87.000000,36.000000,"Higher grow limit","moving_average","lastResponseRTT"
2,1522941686.000000,107.000000,36.000000,"Higher grow limit","moving_average","lastResponseRTT"
2,1522941691.000000,105.200000,36.000000,"Higher grow limit","moving_average","lastResponseRTT"
2,1522941696.000000,98.400000,36.000000,"Higher grow limit","moving_average","lastResponseRTT"
2,1522941701.000000,87.600000,36.000000,"Higher grow limit","moving_average","lastResponseRTT"
